frontend/catalyst/compilation_pipelines.py

```diff
@@ -444,6 +448,97 @@ def __call__(self, *args, **kwargs):
         return result
 
 
+class Directory:
```

Comment:
Should these classes (`Directory`, `WorkspaceManager`) go into utils? That might avoid the `too-many-lines` flag, and also the circular import in the other file.

Reply:
https://github.com/PennyLaneAI/catalyst/pull/306/commits/e4fd4f7dd28d97a082756d389feda2ffeb577e72 Thanks!